### DISCUSSION TITLE
Global ranking graph still exposed in inactive profile.

### DISCUSSION THREAD

**Discussion by Semro** [ppy/osu#16156] [Q&A] [answered]
<details>
  <summary>osu!lazer screenshot:</summary>

![osu_2021-12-19_13-29-20](https://user-images.githubusercontent.com/3332159/146671976-faa0fb55-2f1c-4339-9636-b362e16e3b71.jpg)

</details>
<details>
  <summary>osu!web screenshot:</summary>

![image](https://user-images.githubusercontent.com/3332159/146671989-4168cc33-8234-4a44-930f-2e62b00ca5ed.png)

</details>

osu!lazer version: 2021.1218.0

**Comment by bdach** [comment, marked as answer]
Will be fixed by https://github.com/ppy/osu/pull/16158
